pkg/debug/checkups/system_time.go

```diff
@@ -20,21 +20,22 @@ func (st *systemTime) Name() string {
 func (st *systemTime) Run(_ context.Context, extraFh io.Writer) error {
 	var (
 		urls = []string{
+			"https://app.kolide.com/version",
```

Comment:
```suggestion
			"https://k2control.kolide.com/version",
```
I'm willing to try it, but we have fairly limited concurrency on our servers